backend/src/main/kotlin/app/ehrenamtskarte/backend/application/webservice/EakApplicationMutationService.kt

```diff
@@ -128,4 +129,29 @@ class EakApplicationMutationService {
             }
         }
     }
+
+    @GraphQLDescription("Updates a note of an application")
+    fun updateApplicationNote(
+        applicationId: Int,
+        noteText: String,
```

Comment:
I think we should put *some* restriction on `noteText`, e.g. at most 1000 characters or so.